### DISCUSSION TITLE
IndexHTML

### DISCUSSION THREAD

**Discussion by menzhessarov** [benbjohnson/wtf#44] [Q&A] [answered]
Hello, could you please explain why do you use index.html in assets folder if there is already index.ego?

```
var IndexHTML []byte
```

**Comment by benbjohnson** [comment, marked as answer]
Good question. The `index.html` asset is for the static home page when the user is logged out. The `index.ego` is a template that renders the logged in view. https://github.com/benbjohnson/wtf/blob/63b98f9f3c2da8d7480c867357a225c0d5d7c131/http/server.go#L424-L434
